### PR TITLE
fix: hide uploads manager progress bar for screen-readers if not visible

### PR DESCRIPTION
**Pull request by loonskai**
This PR fixes the issue of `OverallUploadsProgressBar` being accessible for screen readers when it's visually hidden

### CODE REVIEW

File: src/elements/content-uploader/OverallUploadsProgressBar.js [hunk 1]
```diff
@@ -94,6 +94,7 @@ const OverallUploadsProgressBar = ({
             onKeyDown={onKeyDown}
             role="button"
             tabIndex={isVisible ? '0' : '-1'}
+            aria-hidden={!isVisible}
```
Comment (dsporysz-box) [inline src/elements/content-uploader/OverallUploadsProgressBar.js:97]: nit: props should be sorted alphabetically

Comment (loonskai) [reply src/elements/content-uploader/OverallUploadsProgressBar.js:97]: I'm wondering if we should enable `react/jsx-sort-props` rule in this case? This is easy to miss if checked manually

Comment (dsporysz-box) [reply src/elements/content-uploader/OverallUploadsProgressBar.js:97]: I like the idea but I don't what effect it's going to have on BUIE users. I think it's a topic for another issue/PR. 😄

**Review by dsporysz-box — DISMISSED** [review]
LGTM

**Review by dsporysz-box — APPROVED** [review]

**Review by JChan106 — APPROVED** [review]

**Pull request merged**